loopchain/peermanager/peer_manager.py

```diff
@@ -38,11 +26,11 @@ def __init__(self):
         # reps_hash, reps for reset_all_peers
         self._reps_reset_data: Optional[tuple] = None
 
-        self._prepared_reps_hash = None
+        self._crep_root_hash = Hash32.fromhex(conf.CHANNEL_OPTION[ChannelProperty().name].get('crep_root_hash'))
```

Comment:
Suggestion:

I think `PeerManager` needs to channel_name to be initialized. Please consider:
```python
class PeerManager:
    def __init__(self, channel_name):
        ....
        self._crep_root_hash = Hash32.fromhex(conf.CHANNEL_OPTION[channel_name].get('crep_root_hash'))
```
Using Singleton property to initialize `PeerManager` is risky.

I checked that `ChannelService` set its name ahead of `PeerManager` init.
But someday if changed initialization order, which means PeerManager first then ChannelProperty follows, it could be hard to find which line raises error.

Reply:
I understand your worries. But this is not the last form. Peer Manager will be modified to not handle this 'crep_root_hash' in subsequent refactorings.